tutor/src/components/course-calendar/duration.cjsx

```diff
@@ -81,25 +80,17 @@ CourseDuration = React.createClass
       .value()
 
```

Comment:
I'm not sure using the joined array to sort will do what you want.   For instance picture a array of values such as this (with values representing what's returned by the `valueOf`):

| values| | after join()|
|------------|---|--------:|
| `[ 10, 1, 3, 101 ]` | => | `1013111` |
| `[ 11, 1, 3, 99 ]`|   => | `111399` |

The second should be first because it's `published_at` is 11, but it'll come last because the # of digits aren't as long.

The "right" way to do this is to multiple the numbers by their place, then add them.

| values| | equals|
|------------|---|--------:|
| `(10*1000) + (1*100) + (3*10) + 101` | => | `10231` |
| `(11*1000) + (1*100) + (3*10) + 99`|   => | `11229` |